docs/new-architecture-app-modules-android.md

```diff
@@ -18,7 +20,7 @@ You can mitigate this by following the approach described in [Speeding up your B
 
 :::
 
-The code-gen will output some Java and some C++ code that now we need to build.
+The code-gen will output some Java/Kotlin and some C++ code that now we need to build.
```

Comment:
That's not true. The codegen will always output Java code, that can interoperate with your Kotlin code.

Reply:
removed